x-pack/plugins/observability/public/pages/home/index.tsx

```diff
@@ -0,0 +1,258 @@
+      defaultMessage: 'Demo Playground',
+    }),
+    icon: 'play',
+    description: '',
+    href: 'https://demo.elastic.co/',
+    target: '_blank',
+  },
+  {
+    id: 'sampleData',
+    title: i18n.translate('xpack.observability.section.tryItOut.sampleData.title', {
+      defaultMessage: 'Add sample data',
+    }),
+    icon: 'documents',
+    description: '',
+    href: '/app/home#/tutorial_directory/sampleData',
+  },
+];
+
+const Container = styled.div`
+  min-height: calc(100vh - 48px);
+  background: ${(props) => props.theme.eui.euiColorEmptyShade};
+`;
+
+const Title = styled.div`
+  background-color: ${(props) => props.theme.eui.euiPageBackgroundColor};
+  border-bottom: ${(props) => props.theme.eui.euiBorderThin};
+`;
+
+const Page = styled.div`
+  width: 100%;
+  max-width: 1200px;
+  margin: 0 auto;
+  overflow: hidden;
+}
+`;
+
+export const Home = () => {
+  const { core } = usePluginContext();
+
+  useEffect(() => {
+    core.chrome.setBreadcrumbs([
+      {
+        text: i18n.translate('xpack.observability.home.breadcrumb.observability', {
+          defaultMessage: 'Observability',
+        }),
+      },
+      {
+        text: i18n.translate('xpack.observability.home.breadcrumb.gettingStarted', {
+          defaultMessage: 'Getting started',
+        }),
+      },
+    ]);
+  }, [core]);
+
+  return (
+    <Container>
+      <Title>
+        <Page>
+          <EuiSpacer size="xxl" />
+          <EuiFlexGroup>
+            <EuiFlexItem grow={false}>
+              <EuiIcon type="logoObservability" size="xxl" />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiTitle size="m">
+                <h1>
+                  {i18n.translate('xpack.observability.home.title', {
+                    defaultMessage: 'Observability',
+                  })}
+                </h1>
+              </EuiTitle>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="xxl" />
+        </Page>
+      </Title>
+      <Page>
+        <EuiSpacer size="xxl" />
+        <EuiFlexGroup direction="column">
+          {/* title and description */}
+          <EuiFlexItem style={{ maxWidth: '50%' }}>
+            <EuiTitle size="s">
+              <h2>
+                {i18n.translate('xpack.observability.home.sectionTitle', {
+                  defaultMessage: 'Observability built on the Elastic Stack',
+                })}
+              </h2>
+            </EuiTitle>
+            <EuiSpacer size="m" />
+            <EuiText size="s" color="subdued">
+              {i18n.translate('xpack.observability.home.sectionsubtitle', {
+                defaultMessage:
+                  'Bring your logs, metrics, and APM traces together at scale in a single stack so you can monitor and react to events happening anywhere in your environment.',
+              })}
+            </EuiText>
+          </EuiFlexItem>
+
+          {/* Apps sections */}
```

Comment:
We might need to add an option in EUI to render EuiCard without the panel (border & shadow) display. But I'd certainly switch to using the EuiCard, and I can work on getting this display style into EUI

Reply:
I thought about that too, but since we talked in one of the comments above that we shouldn't overwrite the Eui styles I thought I shouldn't do it. 

But I also agree that replacing it by EuiCard is way better.

Since both of you are fine with that, I go with the EuiCard here too.

Reply:
@cauemarcondes thanks for considering that! In most cases, we want to do a flow like this (@sqren please correct me if I'm off here for APM):
1) Use an existing EUI component as-is or with its documented props/options
2) Use an existing EUI component, and ping the EUI team about a special need we have (we may need to hack around it for a release or so but with the idea we will use the EUI recommended way eventually)
3) In rare cases, use our own styles to override EUI styles because what we want can't be accommodated by EUI

I think we ran into a case of (2) here, and the EUI team is the BEST at helping us out with those situations. Thanks for being flexible!

Reply:
Thanks for the clarification @jasonrhodes .

Reply:
@cchaos I just noticed there's a property called `display` available on `EuiCard` which has the options `plain` or `panel` when choosing `plain` it removes the border and shadow.

<img width="650" alt="Screenshot 2020-06-05 at 10 43 46" src="https://user-images.githubusercontent.com/55978943/83856171-73f9e780-a719-11ea-8c20-e09ada773aa8.png">

Reply:
Gah, hahaha, You're right. We just never added an example in our docs specifically for that. Glad you found it!